### PR TITLE
Update jest configuration

### PR DESCRIPTION
**Pull request by chuanxd**
### What? Why?
- add `collectCoverageFrom`

image

refs: http://facebook.github.io/jest/docs/configuration.html#collectcoveragefrom-array
### How was it tested?

npm run test

cc @rwu823 @CApopsicle

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,10 @@
     ],
     "setupFiles": [
       "./test/setup.js"
+    ],
+    "collectCoverageFrom": [
+      "**/src/components/**/*.{js,jsx}",
```
Comment (rwu823) [inline package.json]: `src/components/*/index.js` 只要 check 這隻 main file 就好了，match 速度會較快

**Comment by rwu823** [comment]
Well done

File: package.json [hunk 2]
```diff
@@ -22,6 +22,10 @@
...
+      "!**/src/components/index.js"
```
Comment (rwu823) [inline package.json]: 用上面寫法可以不用這行

**Review by rwu823 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (chuanxd) [reply package.json]: @rwu823 

我本來是這樣寫，但是會錯

<details>

``` sh
Running coverage on untested files...
              Failed to collect coverage from /Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/src/components/index.js
              ERROR: Error: /Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/src/components/index.js: Can't resolve '~dist/owl-ui.css' in '/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/src/components'
              STACK: Error: /Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/src/components/index.js: Can't resolve '~dist/owl-ui.css' in '/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/src/components'
    at onResolved (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/Resolver.js:66:16)
    at loggingCallbackWrapper (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at afterInnerCallback (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/Resolver.js:138:10)
    at loggingCallbackWrapper (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at Resolver.applyPluginsAsyncSeriesBailResult1 (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/tapable/lib/Tapable.js:108:46)
    at innerCallback (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/Resolver.js:125:19)
    at loggingCallbackWrapper (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at /Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/tapable/lib/Tapable.js:210:15
    at innerCallback (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/Resolver.js:123:11)
    at loggingCallbackWrapper (/Users/chuanxd/Documents/cepave/workspace/vue-owl-ui/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
```

</details>

`/src/components/index.js`

``` js
const isNode = !(typeof window !== 'undefined' && window.document && document.createElement)

if (process.env.NODE_ENV === 'build' && !isNode) {
  const style = document.createElement('style')
  style.textContent = require('raw!~dist/owl-ui.css')

  document.head.appendChild(style)
}

module.exports = {
  Select: require('./select')
}
```

**Review by rwu823 — APPROVED** [review]

**Pull request merged**